x-pack/plugins/security_solution/public/management/pages/event_filter/view/components/notification/index.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useState } from 'react';
+
+import { isCreationSuccessful, getFormEntry, getCreationError } from '../../../store/selector';
+
+import { useToasts } from '../../../../../../common/lib/kibana';
+import { useEventFilterSelector } from '../../hooks';
+import { getCreationSuccessMessage, getCreationErrorMessage } from './translations';
+
+export const EventFilterNotification = memo(() => {
```

Comment:
FYI - I know you are probably following the pattern that was done in Trusted Apps by the prior team member. In most cases, I see notifications being done right from the component that needs to show them and not have a sepeerate component that includes its own logic for showing it. Just FYI :)

Reply:
Cool! Yes, I followed the same structure to have both tabs consistent + o - in terms of code structure.